src/content/reference/react/use-client.md

```diff
@@ -350,26 +353,27 @@ export default function Circle() {
   useLayoutEffect(() => {
     const canvas = ref.current;
     const context = canvas.getContext('2d');
+    context.reset();
```

Comment:
effects can run multiple times :) like with offscreen in the future (and also because `[]` deps aren't specified here)

could've put this in the unmount but I put it here